src/components/Paginate.js

```diff
@@ -85,7 +85,7 @@ const Paginate = ({ count, active, urlF, onClick }) => {
   );
 
   return (
-    <Pagination aria-label="Table page" className="mt-3">
+    <Pagination aria-label="Table page" className="mt-3 Pagination">
```

Comment:
this  is `.pagination` in the CSS... not sure if capitalization matters here, but we should lower it just for consistency sake

Reply:
CSS class names are case-sensitive, so this matters.
```suggestion
    <Pagination aria-label="Table page" className="mt-3 pagination">
```